### ISSUE TITLE
Proposal: OTabs: Add animation props (as like as BTabs of Buefy)

### ISSUE THREAD

**Issue by mk-startialab** [oruga-ui/oruga#769]
### Description

I suggest adding animation related props, similar to BTabs in Buefy.

### Why Oruga need this feature

To promote switching from BTabs of Buefy to Otabs.

**Comment by mlmoravek** [comment]
I agree with the `animateInitially` prop. But I'm not sure about the transition name override. How does this work with having for activation and deactivation the same animation?
